src/styles/AddItemForm.css

```diff
@@ -0,0 +1,31 @@
+.form {
+  margin: 0 auto;
+  width: 35%;
```

Comment:
Love all this css! The form is SO beautiful! This `width` being set to `35%` is causing some interesting scaling. But I don't wanna get too hung up on styling at this point. But if you wanna dig in more to this I'm happy to do some brainstorming with you. If not, feel comfy knowing we'll be spending the whole last week on styling!